app/__init__.py

```diff
@@ -9,7 +9,7 @@
 from flask.ext.wtf import CsrfProtect
 from flask.ext.compress import Compress
 from config import config
-from assets import app_css, app_js, vendor_css, vendor_js
+from assets import app_css, app_js, vendor_css, vendor_js, iThing_css, jqueryui_js, jQDateRangeSlidermin_js
```

Comment:
`iThing` is also suuuuuper vague, pls rename to something more specific? what is it?